frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -138,23 +156,29 @@ export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state,
           )}
         </div>
       </div>
-      <ChartDonut
-        ariaDesc={t('ceph-storage-plugin~Selected versus Available Capacity')}
-        ariaTitle={t('ceph-storage-plugin~Selected versus Available Capacity')}
-        height={220}
-        width={220}
-        radius={radius}
-        data={donutData}
-        labels={({ datum }) => `${humanizeBinaryBytes(datum.y).string} ${datum.x}`}
-        subTitle={t('ceph-storage-plugin~Out of {{capacity}}', {
-          capacity: humanizeBinaryBytes(state.chartTotalData).string,
-        })}
-        title={availableCapacityStr}
-        constrainToVisibleArea
-        subTitleComponent={
-          <ChartLabel dy={5} style={{ fill: `var(--pf-global--palette--black-500)` }} />
-        }
-      />
+      {isLoadingDonutChart ? (
+        <div style={{ marginTop: '95%' }}>
```

Comment:
Do not use style element directly

Reply:
@GowthamShanmugam Yes. I will change it.